### DISCUSSION TITLE
Add an option to display most recent media on a side

### DISCUSSION THREAD

**Discussion by RobotoSkunk** [typefully/minimal-twitter#134] [Ideas] [answered]
I follow a few artists on Twitter, and being able to preview the latest images they've posted is very useful to me.

I guess it would be a good idea to put in an option to optionally enable those images.

**Comment by ghost** [comment, marked as answer]
Will be live in next release: https://github.com/typefully/minimal-twitter/pull/141/commits/c1dd8f698b071fd327c8227d5358a978c0a88984
